beacon-chain/core/validators/validator.go

```diff
@@ -131,6 +131,26 @@ func GetShardAndCommitteesForSlot(shardCommittees []*pb.ShardAndCommitteeArray,
 	return shardCommittees[index], nil
 }
 
+// GetBeaconProposerIndex returns the index of the proposer of the block at a
+// given slot.
+//
+// Spec pseudocode definition:
+//  def get_beacon_proposer_index(state: BeaconState,slot: int) -> int:
+//    """
+//    Returns the beacon proposer index for the ``slot``.
+//    """
+//    first_committee = get_shard_committees_at_slot(state, slot)[0].committee
+//    return first_committee[slot % len(first_committee)]
+func GetBeaconProposerIndex(state *pb.BeaconState, slot uint64) (uint32, error) {
```

Comment:
Golang naming conventions prefer not using `Get` as a prefix

Reply:
I was actually following the spec, should we change it then from how it is specified in the spec ?

Reply:
We should change it to conform to go standards